### PR TITLE
Removes Chat components, upgrades dependencies to match HSApp

### PR DESCRIPTION
**Pull request by tinkertrain**
This PR:

- Removes ChatSidebar
- Removes ChatInbox
  - Both moved to HS App: https://github.com/helpscout/hs-app/pull/8613
- Removes ChatTranscript
  - Moved to HS App: https://github.com/helpscout/hs-app/pull/8754
- Cleans up dependencies:
  - Removes a couple of unused dependencies
  - Updates Prettier config file
- Match react and jest versions from: https://github.com/helpscout/hs-app/pull/8176/files#diff-2ac18b788648958ababf0b8169011dfd
  - Jest 26: fixes tests affected with the upgrade
    - Notes:
		- Use `jest.runAllTimers()` instead of `jest.runAllImmediates()` (https://jestjs.io/docs/en/jest-object.html#jestrunallimmediates)
        - `document.activeElement` doesn't work as it used to, so different strategies to test are required (check VerificationCode and Input tests for a couple of ways to test the same)
- Adjusts some `PropTypes` declarations based on HSApp needs

### Testing

- All Tests pass
- Storybook runs without issue
- Builds fine

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -80,12 +80,13 @@
     "make:version": "node ./scripts/make-version.js"
   },
   "peerDependencies": {
-    "react": "^16.8",
-    "react-dom": "^16.8",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
     "styled-components": "^5"
```
Comment (plbabin) [inline package.json:85]: maybe we should target 5.1 ?

**Comment by plbabin** [comment]
We might need to sync with this PR #856 and see if we need to close it or not. Maybe we want to limit the scope to just a react version update first, and then migrate the rest with this PR. Since you did all the work @tinkertrain, I think you can decide what is best moving forward

**Comment by cocoabythefire** [comment]
> We might need to sync with this PR #856 and see if we need to close it or not. Maybe we want to limit the scope to just a react version update first, and then migrate the rest with this PR. Since you did all the work @tinkertrain, I think you can decide what is best moving forward

yeah feel free to close mine - i just threw that up to make sure we wouldn't forget, but it's redundant to these changes here

File: package.json (hunk 1 above)
Comment (tinkertrain) [reply package.json:85]: agree 👍

**Comment by tinkertrain** [comment]
@plbabin @cocoabythefire Woops, I didn't see that PR... React was on `^16.8` which means that it was already installing >16.13.1, so there should not be any impact, I'll close the other PR

**Pull request merged**